js-envs/src/main/scala/org/scalajs/jsenv/nodejs/NodeComJSRunner.scala

```diff
@@ -0,0 +1,183 @@
+/*                     __                                               *\
+**     ________ ___   / /  ___      __ ____  Scala.js sbt plugin        **
+**    / __/ __// _ | / /  / _ | __ / // __/  (c) 2013, LAMP/EPFL        **
+**  __\ \/ /__/ __ |/ /__/ __ |/_// /_\ \    http://scala-js.org/       **
+** /____/\___/_/ |_/____/_/ | |__/ /____/                               **
+**                          |/____/                                     **
+\*                                                                      */
+
+
+package org.scalajs.jsenv.nodejs
+
+import java.io.{Console => _, _}
+import java.net._
+
+import org.scalajs.core.tools.io._
+
+import org.scalajs.jsenv.Utils.OptDeadline
+import org.scalajs.jsenv._
+
+import scala.concurrent.TimeoutException
+import scala.concurrent.duration._
+
+private[nodejs] trait NodeComJSRunner extends ComJSRunner {
```

Comment:
There is bug in the compiler. For some reason if do that it fails with the message `No getter for variable comSocket in trait AbstractNodeJSEnv$NodeComJSRunner` along with it's `scalac` tree.